config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users, path: '/users' do
+    resources :posts, path: '/posts'
+  end
```

Comment:
## [OPTIONAL]

- [ ] Kindy I would suggest you make sure your app has a default root that will be used when the application starts.

```rb
  root 'users#index'
```

Reply:
I didn't know how to do that! Thank you very much :+1: